extensions/amp-form/0.1/test/test-amp-form.js

```diff
@@ -2426,6 +2426,28 @@ describes.repeated(
                 });
               });
 
+              it('should redirect users if header is set but json rejects', () => {
+                sandbox.stub(ampForm.xhr_, 'fetch').returns(
+                  Promise.resolve({
```

Comment:
Let's use `.resolves` here

Reply:
Would prefer to keep as is since that is more self-consistent with other stubs in this file. WDYT?

Reply:
(y) we can switch them on a different PR since this is a large file

Reply:
Follow up PR: #25539